packages/theia-extension/cdtcloud/src/browser/cdtcloud-widget.tsx

```diff
@@ -4,18 +4,33 @@ import { injectable, postConstruct, inject } from '@theia/core/shared/inversify'
 import { AlertMessage } from '@theia/core/lib/browser/widgets/alert-message';
 import { ReactWidget } from '@theia/core/lib/browser/widgets/react-widget';
 import { MessageService } from '@theia/core';
-
+import { CompilationService, DeviceTypeService } from "../common/protocol";
+import { EditorManager } from '@theia/editor/lib/browser/editor-manager';
+import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
 
```

Comment:
Import for comment below

```suggestion
import { FileUri } from "@theia/core/lib/node/file-uri";

 ```